Sources/Build/BuildPlan.swift

```diff
@@ -542,16 +553,28 @@ public final class SwiftTargetBuildDescription {
     /// These are the source files derived from plugins.
     private var pluginDerivedSources: Sources
 
+    /// These are the resource files derived from plugins.
+    private var pluginDerivedResources: [Resource]
+
     /// Path to the bundle generated for this module (if any).
     var bundlePath: AbsolutePath? {
-        buildParameters.bundlePath(for: target)
+        if let bundleName = target.underlyingTarget.potentialBundleName, !resources.isEmpty {
```

Comment:
should we add `target.potentialBundleName` pointing to `target.underlyingTarget.potentialBundleName`?

Reply:
see above